wrappers/gin/gin.go

```diff
@@ -12,18 +18,67 @@ import (
 // TracerKey is the key of the epsagon tracer in the gin.Context Keys map passed to the handlers
 const TracerKey = "EpsagonTracer"
 
+// EpsagonContext creates a context.Background() with epsagon's associated tracer for nexted instrumentations
+func EpsagonContext(c *gin.Context) context.Context {
+	return epsagon.ContextWithTracer(c.Keys[TracerKey].(tracer.Tracer))
+}
+
 // GinRouterWrapper is an epsagon instumentation wrapper for gin.RouterGroup
 type GinRouterWrapper struct {
 	gin.IRouter
 	Hostname string
 	Config   *epsagon.Config
 }
 
+func processRawQuery(urlObj *url.URL, wrapperTracer tracer.Tracer) string {
+	if urlObj == nil {
+		return ""
+	}
+	processed, err := json.Marshal(urlObj.Query())
+	if err != nil {
+		wrapperTracer.AddException(&protocol.Exception{
+			Type:      "trigger-creation",
+			Message:   fmt.Sprintf("Failed to serialize query params %s", urlObj.RawQuery),
+			Traceback: string(debug.Stack()),
+			Time:      tracer.GetTimestamp(),
+		})
```

Comment:
should return here "" (otherwise, if string is nil - string(nil) will panic)

Reply:
:+1: